sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/NestedColumnAliasingSuite.scala

```diff
@@ -336,7 +336,7 @@ class NestedColumnAliasingSuite extends SchemaPruningTest {
 
     val query = companies
       .generate(Explode('employers.getField("company")), outputNames = Seq("company"))
-      .select('company.getField("name"))
+      .select('company.getField("name"), 'company.getField("address"))
```

Comment:
If we want to revise this test case, shall we revise the test name together?

Reply:
okay. revised.